Composer/packages/lib/indexers/src/jsonSchemaFileIndexer.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { FileInfo, JsonSchemaFile } from '@bfc/shared';
+import has from 'lodash/has';
+
+import { getBaseName } from './utils/help';
+
+const index = (jsonFiles: FileInfo[]) => {
+  return jsonFiles.reduce((jsonSchemaFiles: JsonSchemaFile[], { content, name }) => {
```

Comment:
since the input type is an array of fileinfo, that can be any file basically, it's better to filter for the interesting extension (i.e. .json) within the body of index.

It makes sure that the JSON.parse doesn't blow up

Reply:
Agree, but none of the other indexers do that (yet).  I guess to allow for indexing multiple extensions that happen to contain JSON.